### PR TITLE
Contribs rename

### PR DESCRIPTION
**Pull request by aethelwulffe**
Ref #979 
Moved all code packages to
modules/

Corrected urls.

Corrected documentation / comment references where practical.

Renamed directories with utility packages that are not currently called by anything directly (including the menu builder tool and Perl utilities) to util_* as a identification standard.

Suggest moving the util_* features to the Contribs repository.

Note:  The log recovery Perl tool does not work, and is not IAW current scheme, and there is a better way to do that now.  I removed it.

the "dupecheck" feature is not called by anything, nor is it in the menu, and I am not sure if it works.  Same for the de-identification script.  These may require refactoring, removal, or relocation to the contribs repo.

### CODE REVIEW

File: interface/fax/fax_dispatch.php [hunk 1]
```diff
@@ -200,8 +200,7 @@ function mergeTiffs() {
       }
 
       if (!$info_msg) {
-        // The following is cloned from contrib/forms/scanned_notes/new.php:
-        //
+// scanned notes must be installed, and does not natively exist.
```
Comment (aethelwulffe) [inline interface/fax/fax_dispatch.php]: Without re-creating the scanned forms code, this does not exist and is a bad reference.  The fax tool here is probably legacy poop anyway.

Comment (tmccormi) [reply interface/fax/fax_dispatch.php]: The version that bowen has is more generic and useful.  Perhaps we should pull that in to another P/R

**Comment by aethelwulffe** [comment]
FAX code?

**Comment by tmccormi** [comment]
Yes interface/fax   I modified it to be used for a directory tree of incoming scanned documents (PDF) and dropped all the TIFF stuff.

Needs cleanup and maybe renaming to scan* instead of fax ..  I can push a branch with it.

Will merge this in the meantime

**Pull request merged**